apps/src/lib/kits/maker/ui/SetupChecklist.jsx

```diff
@@ -225,27 +225,44 @@ export default class SetupChecklist extends Component {
   }
 
   installFirmwareSketch() {
-    let firmataFromBoardType;
-    switch (this.state.boardTypeDetected) {
-      case BOARD_TYPE.EXPRESS:
-        firmataFromBoardType =
-          'https://learn.adafruit.com/adafruit-circuit-playground-express/code-org-csd';
-        break;
-      case BOARD_TYPE.MICROBIT:
-        firmataFromBoardType =
-          'https://github.com/microbit-foundation/microbit-firmata#installing-firmata-on-your-bbc-microbit';
-        break;
-      default:
-        firmataFromBoardType =
-          'https://learn.adafruit.com/circuit-playground-firmata/overview';
+    let firmataMarkdown;
+    if (this.state.boardTypeDetected === BOARD_TYPE.MICROBIT) {
```

Comment:
this conditional is difficult to read because it has a lot of repetitive info. we're also referencing hard-coded URLs multiple times, which could lead to issues in the future where we update the URL in one place but not the others. i think we should pull the URLs into constants and also set variables for info that's accessed multiple times, like `this.state.boardTypeDetected`